pyb4model/pyb4model.py

```diff
@@ -1,3 +1,8 @@
+import numpy as  np
+import pandas as pd
+import pytest
+from sklearn.model_selection import cross_val_score
```

Comment:
- add your dependencies to the .toml file by typing in your terminal `poetry add package` ! or there will be an error creating the package
- see tiffany's tutorial for reference! https://ubc-mds.github.io/py-pkgs/whole-game.html#pkg-func-deps

Reply:
these dependecies have been added by xinwen